test/cli/autogen-autoloader/autogen-autoloader.out

```diff
@@ -0,0 +1,204 @@
+No errors! Great job.
```

Comment:
Does this provide good enough test coverage?
Are you comfortable that we won't break you as long as this test passes?